src/components/elections-graph/elections-graph.stories.js

```diff
@@ -0,0 +1,8 @@
+import "./elections-graph.component.js";
+
+export default {
+  title: "Components/Elections Graph",
+  component: "elections-graph",
+};
+
+export const Default = {};
```

Comment:
Can we add a default story for this? Maybe just the title, ID and then some dummy data?